grpc/resolver.go

```diff
@@ -0,0 +1,105 @@
+package grpc
+
+import (
+	"errors"
+	"fmt"
+	"net"
+	"strings"
+
+	"google.golang.org/grpc/resolver"
+)
+
+// init registers the `staticBuilder` with the gRPC resolver registry.
+func init() {
+	resolver.Register(NewStaticBuilder())
+}
+
+// NewStaticBuilder creates a `staticBuilder` used to construct static DNS
+// resolvers.
+func NewStaticBuilder() resolver.Builder {
```

Comment:
Do `NewStaticBuilder` and `NewStaticResolver` need to be exported? I believe they're only called from this file.

Reply:
You know, I think you're correct here. I was riffing from the existing internal resolvers (`dns` and `passthrough`), but I don't believe they need to be exported.